pages/resources/quests.mdx

```diff
@@ -0,0 +1,329 @@
+      "secondary": "#5318A7"
+    },
+    "assets": {
+      "reward_tile": "reward.webm",
+      "hero": "hero.png",
+      "quest_bar_hero": "quests_bar_hero.png",
+      "game_tile": "game_tile.png",
+      "logotype": "game_logotype.svg"
+    }
+  },
+  "user_status": {
+    "user_id": "222069018507345921",
+    "quest_id": "8206816794116096000",
+    "enrolled_at": "2077-01-01T11:59:59+00:00",
+    "completed_at": "2077-01-01T11:59:59+00:00",
+    "claimed_at": "2077-01-01T11:59:59+00:00",
+    "claimed_tier": null,
+    "last_stream_heartbeat_at": "2077-01-01T11:59:59+00:00",
+    "stream_progress_seconds": 1800,
+    "dismissed_quest_content": 0,
+    "progress": {}
+  }
+}
+```
+
+### Quest User Status Object
+
+Represents the user's quest progression.
+
+###### Quest User Status Structure
+
+| Field                     | Type              | Description                                                                                                                     |
+| ------------------------- | ----------------- | ------------------------------------------------------------------------------------------------------------------------------- |
+| user_id                   | snowflake         | The user's ID                                                                                                                   |
+| quest_id                  | snowflake         | The quest's ID                                                                                                                  |
+| enrolled_at               | ISO8601 timestamp | When the user accepted the quest                                                                                                |
+| completed_at              | ISO8601 timestamp | When the user completed the quest                                                                                               |
+| claimed_at                | ISO8601 timestamp | When the user claimed the quest's reward                                                                                        |
+| claimed_tier              | unknown           | Unused                                                                                                                          |
+| last_stream_heartbeat_ack | ISO8601 timestamp | Timestamp of the latest stream heartbeat request                                                                                |
+| stream_progress_seconds   | ISO8601 timestamp | The total amount of seconds the user has been streaming the game for since the quest started and before the quest was completed |
+| dismissed_quest_content   | ISO8601 timestamp | Whenever the user has dismissed the quest                                                                                       |
+| progress                  | object            | Unused object that could have been an early iteration of the user status object                                                 |
```

Comment:
Debating whether this should just be removed, since apparently it's a holdout from drops. Not sure how I feel about documenting unknown stuff. It's probably fine for now :p